randomteleport-plugin/pom.xml

```diff
@@ -42,6 +42,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}.pluginhook</groupId>
+            <artifactId>chunkyborder</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
You seem to have forgotten to add the GriefDefender hook atifact here.

Reply:
Oops, added it in.